res/terminal/Terminal.svg

```diff
@@ -1,89 +1 @@
 result="shape"/></filter><filter id="filter1_dd" width="18.95" height="18.875" x="3.025" y="15" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy=".5"/><feGaussianBlur stdDeviation=".5"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0"/><feBlend in2="BackgroundImageFix" mode="normal" result="effect1_dropShadow"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.2 0"/><feBlend in2="effect1_dropShadow" mode="normal" result="effect2_dropShadow"/><feBlend in="SourceGraphic" in2="effect2_dropShadow" mode="normal" result="shape"/></filter><filter id="filter2_dd" width="24" height="11" x="20" y="30" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy=".5"/><feGaussianBlur stdDeviation=".5"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0"/><feBlend in2="BackgroundImageFix" mode="normal" result="effect1_dropShadow"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="1.5"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.2 0"/><feBlend in2="effect1_dropShadow" mode="normal" result="effect2_dropShadow"/><feBlend in="SourceGraphic" in2="effect2_dropShadow" mode="normal" result="shape"/></filter><linearGradient id="paint0_linear" x1="36.446" x2="11.822" y1="47.826" y2="5.175" gradientUnits="userSpaceOnUse"><stop stop-color="#333"/><stop offset="1" stop-color="#4D4D4D"/></linearGradient><linearGradient id="paint1_linear" x1="14.528" x2="10.484" y1="33.996" y2="26.992" gradientUnits="userSpaceOnUse"><stop stop-color="#999"/><stop offset="1" stop-color="#B3B3B3"/></linearGradient><linearGradient id="paint2_linear" x1="14.528" x2="10.484" y1="33.996" y2="26.992" gradientUnits="userSpaceOnUse"><stop stop-color="#999"/><stop offset="1" stop-color="#B3B3B3"/></linearGradient><linearGradient id="paint3_linear" x1="16.275" x2="8.737" y1="30.034" y2="16.978" gradientUnits="userSpaceOnUse"><stop stop-color="#CCC"/><stop offset="1" stop-color="#E6E6E6"/></linearGradient><linearGradient id="paint4_linear" x1="16.275" x2="8.737" y1="30.034" y2="16.978" gradientUnits="userSpaceOnUse"><stop stop-color="#CCC"/><stop offset="1" stop-color="#E6E6E6"/></linearGradient><linearGradient id="paint5_linear" x1="35.15" x2="28.85" y1="39.955" y2="29.045" gradientUnits="userSpaceOnUse"><stop stop-color="#CCC"/><stop offset="1" stop-color="#E6E6E6"/></linearGradient></defs></svg>
```

Comment:
since the SVGs are reference assets that do not ship with the product, I'd prefer they be kept un-minified. Keeping them minified will make it more difficult to diff them if they ever change.

Reply:
Ok I'll revert those ones.

Reply:
I've reverted them and verified they no longer show in the list of changes.